CMakeLists.txt

```diff
@@ -26,8 +26,8 @@ set (CPACK_PACKAGE_VERSION_MINOR ${GLOG_MINOR_VERSION})
 set (CPACK_PACKAGE_VERSION_PATCH ${GLOG_PATCH_VERSION})
 set (CPACK_PACKAGE_VERSION ${GLOG_VERSION})
 
-option (WITH_GFLAGS "Use gflags" ON)
-option (WITH_THREADS "Enable multithreading support" ON)
+option (WITH_GFLAGS "Use gflags" OFF)
+option (WITH_THREADS "Enable multithreading support" OFF)
```

Comment:
Please do not change the defaults. Gflags and Threads are both optional features, even if the `WITH_*` options are set to `ON`. The messages you see during generation are not errors, but warnings.